components/Embed.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface Props {
+  src: string;
+  title?: string;
+  width?: number;
+  height?: number;
+}
+
+export function Embed({ src, title, width, height }: Props) {
+  return (
+    <iframe
+      referrerPolicy="no-referrer"
+      sandbox="allow-forms allow-scripts"
+      src={src}
+      title={title}
+      width={width ?? "100%"}
+      height={height ?? "300px"}
```

Comment:
**issue:** same problem here, we don't want to force user writing more than necessary, which `px` is.

Reply:
So, we leave it hardcoded?
No input from user?

Reply:
No. The problem is, that we ask user for `number`, but then we add it as a number, not as `${number}px`.